modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartQuickAdd.js

```diff
@@ -71,31 +71,80 @@ export default function CartQuickAdd() {
 		fetch(productsApiURL.toString())
 			.then((response) => response.json())
 			.then((availableProducts) => {
-				setFormattedProducts(
-					availableProducts.items.map((product) => {
-						const {name, skus} = product;
-
-						return {
+				const formattedProducts = [];
+
+				availableProducts.items.map((product) => {
```

Comment:
It is a little strange to use `.map` but not set it to value. Since you are filtering this list into `formattedProducts`, you should use `.reduce` instead or swap to `.forEach` since you aren't using the value `.map` returns.